### PR TITLE
[OSCD] Atomic Red Team: Detected Windows Software Discovery (T1518) 

### PR DESCRIPTION
**Pull request by NikitaStormwind**
#1013 Add Rule Task 17

### CODE REVIEW

File: rules/windows/builtin/win_software_discovery.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+action: global
+title: Detected Windows Software Discovery
+id: 2650dd1a-eb2a-412d-ac36-83f06c4f2282
+description: Adversaries may attempt to enumerate software for a variety of reasons, such as figuring out what security measures are present or if the compromised system has a version of software that is vulnerable.
+status: experimental
+author: Nikita Nazarov, oscd.community
+date: 2020/10/16
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1518/T1518.md
+    - https://attack.mitre.org/techniques/T1518/
```
[the remark this answers is not in the document]
Comment (NikitaStormwind) [reply rules/windows/builtin/win_software_discovery.yml]: Hi, @yugoslavskiy !
Done

File: rules/windows/builtin/win_software_discovery.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+---
+logsource:
+    product: windows
+    service: powershell
+detection:
+    selection:
+        EventID: 4104
+        ScriptBlockText|contains|all:    # Example: Get-ItemProperty HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\* | Select-Object DisplayName, DisplayVersion, Publisher, InstallDate | Format-Table -Autosize
+          - 'get-itemProperty'
+          - '*\software\*'
```
[the remark this answers is not in the document]
Comment (NikitaStormwind) [reply rules/windows/builtin/win_software_discovery.yml]: Done

File: rules/windows/builtin/win_software_discovery.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+---
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        Image|endswith: '\reg.exe'    # Example: reg query "HKEY_LOCAL_MACHINE\Software\Microsoft\Internet Explorer" /v svcVersion
+        CommandLine|contains|all:
+            - 'query'
+            - '*\software\*'
```
[the remark this answers is not in the document]
Comment (NikitaStormwind) [reply rules/windows/builtin/win_software_discovery.yml]: Done

**Comment by yugoslavskiy** [comment]
Thank you very much for your contribution, @NikitaStormwind!

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Pull request merged**